### ISSUE TITLE
Import data from other services

### ISSUE THREAD

**Issue by xplosionmind** [ihucos/counter.dev#64]
Rather than developing direct migration from Google Analytics, Plausible, or other [counter.dev alternatives](), I believe that in order to allow users to move to counter.dev it is very, very important to provide them the possibility of **importing data from other analytics** services **via CSV**.

**Comment by ihucos** [comment]
Interesting thought. Is it possible to export the data as csv from google analytics, Plausible and others?

**Comment by xplosionmind** [comment]
> Is it possible to export the data as csv from google analytics, Plausible and others?

It is.

**Comment by ihucos** [comment]
Yep, would actually be a really nice feature. Don't have the time to implement though, it's a tragedy :sweat_smile:
